primeNumber.py

```diff
@@ -0,0 +1,19 @@
+x = int(input("Enter a Number to check:"))  #Taking input from user to check whether given number is prime or not
+if x < 1 :
+    print("Invalid input")
+else:
+    count = 0
```

Comment:
Please change the variable name to isPrimeNumber and make it boolean

Reply:
Done